scheduler/cups-driverd.cxx

```diff
@@ -113,11 +115,17 @@ typedef union				/**** TAR record format ****/
   }	header;
 } tar_rec_t;
 
+struct args {
+    cups_file_t *fp;
+    char *buf;
+    size_t buflen;
+};
```

Comment:
In CUPS we use ```typedef``` for defining structures, see [coding guidelines](https://github.com/OpenPrinting/cups/blob/master/DEVELOPING.md) regarding types.
Other thing is a formatting - CUPS uses spaces till 7 characters, but tab(s) must be used instead of 8 spaces, 16 spaces etc. (so f.e. if you do the indentation for 10 spaces, you will use 1 tab and 2 spaces...).